app/views/admin/questions/_question.html.slim

```diff
@@ -1,20 +1,21 @@
 -@questions.each do |question|
   tr.info
     td
-      =link_to admin_question_path(question)
+      = link_to admin_question_path(question)
         = question.id
     td
-      =link_to admin_topic_path(question.topic_id)
+      = link_to admin_topic_path(question.topic_id)
```

Comment:
I tried a lot of different ways.
_level partial is rendered in _question partial and in question/:id
 _question partial is rendered in questions/index and topic/:id
When _level is rendered in question/:id it takes @question variable from questions_controller 
'''ruby
def show
    @question = Question.find(params[:id])
'''
When _level is rendered in questions/index or topic/:id in each td, it is nowhere to get variable @question from. And we can not define each variable @question for each td in  QuestionsController index method and TopicsController show method.
That's why I can't find another way to do this.